contents/docs/self-host/runbook/clickhouse/resize-disk.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Resize disk
+sidebar: Docs
+showTitle: true
+---
+
+#### How to increase disk size
```

Comment:
Is this runbook slightly different than the one for Kafka/PostgreSQL? The wording seems different and I don't see the prerequisite section

Reply:
It's different as you don't need to do anything other than change the values file

Reply:
Ok but prerequisite should still be here

Reply:
I assumed it's always expandable because of this https://github.com/PostHog/charts-clickhouse/blob/main/charts/posthog/templates/clickhouse_instance.yaml#L15 no?

Reply:
That's technically not enough (it maybe works for AWS and GCP but what about the other clouds?)